src/styles/base.scss

```diff
@@ -131,5 +147,16 @@ ul {
 }
 
 .container-top{
-  padding-top: 80px;
+  padding-top: 10%;
 }
+
+@media(min-width:810px) and (max-width:1130px){
+  .container-top{
+    padding-top:20%;
```

Comment:
Don't give padding in percentage here. @live-wire Use pixel.

Reply:
Done!